Automattic/jetpack
# Update/window state calls with new function

## Proposed changes:

* Replace state calls from the window with a util wrapper function

### Other information:


## Jetpack product discussion

N/A

## Does this pull request change what data or activity we track or use?

No

## Testing instructions:

There are a lot of places where we are replacing the window state call with the new function, what I liked to do was to console.log the values and just make sure they match `window.myJetpackInitialState`. 

1. Checkout this branch via the Jetpack Beta plugin or your local dev environment
2. Make sure My Jetpack loads everything correctly (boost scores, product data, active status)
![image](https://github.com/Automattic/jetpack/assets/65001528/1e5863eb-effb-49bc-a96a-66910de37f9c)

3. Make sure there are no errors in the console
4. Go to `/wp-admin/admin.php?page=my-jetpack#/add-search` and `/wp-admin/admin.php?page=my-jetpack#/add-protect` to make sure everything loads correctly on the product interstitial cards and tables`
5. Make sure the `Manage Your Plan` or `Purchase a plan` link goes to the right place in the Plans section
![image](https://github.com/Automattic/jetpack/assets/65001528/28bc238b-ff02-4d3d-a61f-148293ff7a81)
6. Make sure `/wp-admin/admin.php?page=my-jetpack#/add-license` loads correctly
